Buildings/Fluid/AirFilters/Empirical.mo

```diff
@@ -0,0 +1,304 @@
+        Line(
+          points={{28,2},{44,12},{60,0},{70,10}},
+          color={28,108,200},
+          thickness=0.5),
+        Line(
+          points={{70,10},{60,10}},
+          color={28,108,200},
+          thickness=0.5),
+        Line(
+          points={{70,10},{70,2}},
+          color={28,108,200},
+          thickness=0.5),
+        Line(
+          points={{28,-46},{44,-36},{60,-48},{70,-38}},
+          color={28,108,200},
+          thickness=0.5),
+        Line(
+          points={{70,-38},{60,-38}},
+          color={28,108,200},
+          thickness=0.5),
+        Line(
+          points={{70,-38},{70,-46}},
+          color={28,108,200},
+          thickness=0.5),
+        Ellipse(
+          extent={{-54,70},{-48,64}},
+          lineColor={0,0,0},
+          lineThickness=0.5,
+          fillColor={0,0,0},
+          fillPattern=FillPattern.Solid),
+        Ellipse(
+          extent={{-62,46},{-56,40}},
+          lineColor={0,0,0},
+          lineThickness=0.5,
+          fillColor={0,0,0},
+          fillPattern=FillPattern.Solid),
+        Ellipse(
+          extent={{-50,36},{-44,30}},
+          lineColor={0,0,0},
+          lineThickness=0.5,
+          fillColor={0,0,0},
+          fillPattern=FillPattern.Solid),
+        Ellipse(
+          extent={{-64,24},{-58,18}},
+          lineColor={0,0,0},
+          lineThickness=0.5,
+          fillColor={0,0,0},
+          fillPattern=FillPattern.Solid),
+        Ellipse(
+          extent={{-62,-2},{-56,-8}},
+          lineColor={0,0,0},
+          lineThickness=0.5,
+          fillColor={0,0,0},
+          fillPattern=FillPattern.Solid),
+        Ellipse(
+          extent={{-46,-12},{-40,-18}},
+          lineColor={0,0,0},
+          lineThickness=0.5,
+          fillColor={0,0,0},
+          fillPattern=FillPattern.Solid),
+        Ellipse(
+          extent={{-64,-26},{-58,-32}},
+          lineColor={0,0,0},
+          lineThickness=0.5,
+          fillColor={0,0,0},
+          fillPattern=FillPattern.Solid),
+        Ellipse(
+          extent={{-48,-30},{-42,-36}},
+          lineColor={0,0,0},
+          lineThickness=0.5,
+          fillColor={0,0,0},
+          fillPattern=FillPattern.Solid),
+        Ellipse(
+          extent={{-60,-52},{-54,-58}},
+          lineColor={0,0,0},
+          lineThickness=0.5,
+          fillColor={0,0,0},
+          fillPattern=FillPattern.Solid),
+        Ellipse(
+          extent={{38,38},{44,32}},
+          lineColor={0,0,0},
+          lineThickness=0.5,
+          fillColor={0,0,0},
+          fillPattern=FillPattern.Solid),
+        Ellipse(
+          extent={{40,-16},{46,-22}},
+          lineColor={0,0,0},
+          lineThickness=0.5,
+          fillColor={0,0,0},
+          fillPattern=FillPattern.Solid),
+        Text(
+          extent={{-145,-100},{155,-140}},
+          textColor={0,0,255},
+          textString="%name")}),
+Diagram(coordinateSystem(preserveAspectRatio=false)),
+Documentation(info="<html>
```

Comment:
@SenHuang19 
Do we have any reference for the development? If yes, it's better to add it in the documentation.